system/http.go

```diff
@@ -30,22 +30,23 @@ type HTTP interface {
 }
 
 type DefHTTP struct {
-	http              string
-	allowInsecure     bool
-	noFollowRedirects bool
-	resp              *http.Response
-	RequestHeader     http.Header
-	RequestBody       string
-	Timeout           int
-	loaded            bool
-	err               error
-	Username          string
-	Password          string
-	CAFile            string
-	CertFile          string
-	KeyFile           string
-	Method            string
-	Proxy             string
+	http               string
+	allowInsecure      bool
+	noFollowRedirects  bool
+	resp               *http.Response
+	RequestHeader      http.Header
+	RequestBody        string
+	RequestQueryParams map[string]string
```

Comment:
Should this be: `map[string][]string` to support duplicate query params?

example: `?foo=1&foo=2`

Reply:
Good idea, I'll make the modification